adapter_pipelines/ss2_single_sample/adapter_example_static_demo.json

```diff
@@ -0,0 +1,17 @@
+{
+  "AdapterSs2RsemSingleSample.gtf": "gs://hca-dcp-mint-test-data/reference/demo/gencodev19_chr21.gtf",
+  "AdapterSs2RsemSingleSample.ref_fasta": "gs://hca-dcp-mint-test-data/reference/demo/chr21.fa",
+  "AdapterSs2RsemSingleSample.ref_flat": "gs://hca-dcp-mint-test-data/reference/demo/refFlat.chr21.txt",
+  "AdapterSs2RsemSingleSample.rrna_interval": "gs://hca-dcp-mint-test-data/reference/demo/hg19.chr21.rRNA.interval_list",
+  "AdapterSs2RsemSingleSample.star_genome": "gs://hca-dcp-mint-test-data/reference/demo/star.tar",
+  "AdapterSs2RsemSingleSample.rsem_genome": "gs://hca-dcp-mint-test-data/reference/demo/rsem.tar",
+  "AdapterSs2RsemSingleSample.reference_bundle": "bf51d668-3e14-4843-9bc7-5d676fdf0e01",
+  "AdapterSs2RsemSingleSample.format_map": "gs://hca-dcp-mint-test-data/adapters/file_format_map.json",
+  "AdapterSs2RsemSingleSample.submit_url": "http://api.ingest.staging.data.humancellatlas.org/",
+  "AdapterSs2RsemSingleSample.dss_url": "https://dss.staging.data.humancellatlas.org/v1",
+  "AdapterSs2RsemSingleSample.method": "Ss2RsemSingleSample",
```

Comment:
I think this should be `SmartSeq2SingleCell` like it is in `ss2_single_sample/adapter_example_static.json`